### DISCUSSION TITLE
How to change the Congratulations page

### DISCUSSION THREAD

**Discussion by ZuluPilgrim** [NginxProxyManager/nginx-proxy-manager#3601] [General]
If people visit my external IP, they get 

`Congratulations!
You've successfully started the Nginx Proxy Manager.

If you're seeing this site then you're trying to access a host that isn't set up yet.

Log in to the Admin panel to get started.`

What file do I access to change this?

**Comment by kawanuaid** [comment]
> If people visit my external IP, they get
> 
> `Congratulations! You've successfully started the Nginx Proxy Manager.
> 
> If you're seeing this site then you're trying to access a host that isn't set up yet.
> 
> Log in to the Admin panel to get started.`
> 
> What file do I access to change this?

1. Login to NPM
2. Go to Settings
3. At the Default Site, hit the vertical three-dots, then click Edit.
4. Choose Custom Page, enter your codes to HTML Content.
